python/ray/dashboard/modules/job/job_manager.py

```diff
@@ -84,23 +84,6 @@ def __init__(self, gcs_aio_client: GcsAioClient, logs_dir: str):
         self._recover_running_jobs_event = asyncio.Event()
         run_background_task(self._recover_running_jobs())
 
-    def _get_job_driver_logger(self, job_id: str) -> logging.Logger:
-        """Return job driver logger to log messages to the job driver log file.
-
-        If this function is called for the first time, configure the logger.
-        """
-        job_driver_logger = logging.getLogger(f"{__name__}.driver-{job_id}")
```

Comment:
I kinda feel the right solution is maybe just have a fixed logger to log it like
```
job_driver_logger = logging.getLogger("ray.job.driver")
```

Reply:
(and some kind of dummy path to log those messages)

We can update the message to include the submission id

Reply:
the issue is that this logger wants to write to a file, and that file is never closed once it is open. So although we use 1 logger, we should keep changing files to write. 

I will just keep this as is, but just close the fd as soon as we write this line of log

Reply:
I'm just worried if in the future, we are using this logger to log some other info, we might find the file is already closed and this logger being unusable. But if you think close the fd after each log statement is easier to implement, then I'm also fine with it :)

Reply:
let me think about it a little more about the good solution.